vale/styles/jargonLint/Technology.yml

```diff
@@ -0,0 +1,325 @@
+  OpenAPI: OpenAPI
+  OpenID[ -]?Connect: OpenID Connect
+  OpenSSL: OpenSSL
+  OSM: OSM
+  OSS: OSS
+  OVH(?!\.(?:com|net)): OVH
+  P2P: P2P
+  PaaS: PaaS
+  Parcel: Parcel
+  PayPal(?!\.com): PayPal
+  (?<!\.)PDF: PDF  # also a file extension
+  Perconn?a: Percona
+  Perl: Perl
+  (?<!\.)PHP: PHP  # also a file extension
+  Pinterest(?!\.com): Pinterest
+  Playstation: Playstation
+  (?<!\.)PNG: PNG  # also a file extension
+  pnpm: pnpm
+  Postgres? ?SQL: PostgreSQL
+  Postman: Postman
+  PostScript: PostScript
+  Prettier: Prettier
+  Prometheus: Prometheus
+  Pwned: Pwned
+  RabbitMQ: RabbitMQ
+  React Bootstrap: React Bootstrap
+  React Hook Form: React Hook Form
+  React[ \.]js: React
+  React Native: React Native
+  React: React
+  React-Toastify: React-Toastify
+  READ[ -]?ME(?:s): README
+  Reddit(?!\.com): Reddit
+  Redis: Redis
+  Redux: Redux
+  Redux-Saga: Redux-Saga
+  Redux Toolkit: Redux Toolkit
+  RefluxJS: RefluxJS
+  Renren: Renren
+  RESTful: RESTful
+  REST: REST
+  RFC: RFC
+  RSS: RSS
+  RubyGem: RubyGem
+  RubyGems: RubyGems
+  Rust: Rust
+  SaaS: SaaS
+  Sales?[ -]forces?: Salesforce
+  Samba: Samba
+  SAML: SAML
+  Sass: Sass
+  ScyllaDB: ScyllaDB
+  SDK: SDK
+  SemVer: SemVer
+  SEO: SEO
+  SFTP: SFTP
+  Shopify: Shopify
+  SinaWeibo: SinaWeibo
+  Socialgist: Socialgist
+  Socket.IO: Socket.IO
+  SolidJS: SolidJS
+  SPA: SPA
+  (?<!\.)SQLite3: SQLite3  # also a file extension
+  (?<!\.)SQLite: SQLite  # also a file extension
+  SQLServer: SQLServer
+  (?<!\.)SQL: SQL  # also a file extension
+  SSH: SSH
+  SSL: SSL
+  Stack[ ]?Overflow: Stack Overflow
+  Sublime Text: Sublime Text
+  SUSE: SUSE
+  (?<!\.)SVG: SVG  # also a file extension
+  Symfony: Symfony
+  Tailwind CSS: Tailwind CSS
+  Tamagui: Tamagui
+  TensorFlow.js: TensorFlow.js
+  Tesla: Tesla
+  Three.js: Three.js
+  TikTok(?!\.com): TikTok
+  TODO: TODO
+  (?<!\.)TOML: TOML  # also a file extension
+  Travis ?CI: Travis CI
+  Trustpilot: Trustpilot
+  TSLint: TSLint
+  (?<!\.)TSV: TSV  # also a file extension
+  TTL: TTL
+  TTY: TTY
+  tvOS: tvOS
+  TV: TV
+  Twitter(?!\.com): Twitter
+  TypeScript: TypeScript
+  Ubuntu: Ubuntu
+  UML: UML
+  Unicode: Unicode
+  Unix: Unix
+  upload\.js: upload.js
+  URIs: URIs
+  URI: URI
+  (?<![]"\.])URL: URL  # also a file extension, but JSON with quotes
+  URLs: URLs
+  UUIDs: UUIDs
+  Viewer\.js: Viewer.js
+  VIM: VIM
+  Visual ?Studio ?Code: Visual Studio Code
+  VKontakte: VKontakte
+  VS[- ]?Code: VS Code
+  Vue\.js: Vue.js
+  W3C: W3C
+  watchOS: watchOS
+  WebAssembly: WebAssembly
+  WebKit: WebKit
+  Webpack: Webpack
+  WebStorm: WebStorm
+  Wechat: Wechat
+  Weibo: Weibo
+  Wi[ -]?Fi: Wi-Fi
+  Wikibase: Wikibase
+  Wikidata: Wikidata
+  Wikipedia(?!\.org): Wikipedia
+  Wikitext: Wikitext
+  Windows: Windows
+  WordPress(?!\.com): WordPress
+  WWDC: WWDC
+  Xcode: Xcode
+  Xiaohongshu: Xiaohongshu
+  Xiaomi: Xiaomi
+  (?<!\.)XML: XML  # also a file extension
+  (?<!\.)YAML: YAML  # also a file extension
```

Comment:
Hey @sagepin,

I saw you fixed your yaml pattern detection rule?

https://github.com/sapegin/textlint-rule-terminology/pull/77#discussion_r1656572029

Did my project/PR inspired you?

Reply:
discussion currently in progress about the fact `-` was added to terminology regexp

Reply:
Let's merge like the following ticket will address the possible issue

- #24